src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs

```diff
@@ -54,6 +53,8 @@ public enum DatabaseRecordItemType
         SinkPullReplications = 1 << 13,
         HubPullReplications = 1 << 14,
         TimeSeries = 1 << 15,
-        DocumentsCompression = 1 << 16
+        DocumentsCompression = 1 << 16,
+        IncludeHubReplicationAccessInfo = 1 << 17,
```

Comment:
Don't we need other changes in server side code to handle that? I can see this is only cs file touched in this PR.

Reply:
ok answer is here: https://issues.hibernatingrhinos.com/issue/RavenDB-15680#focus=Comments-67-51338.0-0

Reply:
Why the name contains `Include...` ?